utils/utils.go

```diff
@@ -367,3 +369,56 @@ func GetLocalIP() (string, error) {
 func GetFuncName(fn interface{}) string {
 	return runtime.FuncForPC(reflect.ValueOf(fn).Pointer()).Name()
 }
+
+func WritePidToFile(pid int, path string) error {
```

Comment:
I did see some etcd code using similar code for process management. So, decided to move it utils so that any package can use it.